types/pubsub-js/pubsub-js-tests.ts

```diff
@@ -91,3 +91,21 @@ function ClearAllSubscriptions() {
     // unsubscribe from all subscrpitions
     PubSub.clearAllSubscriptions();
 }
+
+function test_generics_1() {
+    const login = Symbol('LOGIN');
+
+    const pubsub: PubSubJS.Base<number> = PubSub;
+
+    pubsub.subscribe(login, (msg, data) => {});
+
+    pubsub.publish(login, 123);
```

Comment:
```suggestion
    const LOGIN = Symbol('LOGIN');

    const pubsub: PubSubJS.Base<number> = PubSub;

    pubsub.subscribe(LOGIN, (msg, data) => {});

    pubsub.publish(LOGIN, 123);
```

Reply:
Let's keep common code style for entire project.